runtime/isolated-vm/package.json

```diff
@@ -54,5 +54,5 @@
     "test": "node test.js || nodejs test.js"
   },
   "types": "isolated-vm.d.ts",
-  "version": "4.3.6"
+  "version": "4.3.5"
```

Comment:
Looks like this is downgrading the bump from https://github.com/coda/packs-sdk/pull/1583?

Reply:
thanks for catching. will revert.